samples/local.settings.json

```diff
@@ -3,6 +3,8 @@
   "Values": {
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-    "SqlConnectionString": ""
+    "SqlConnectionString": "",
+    "Sp_SelectCost": "SelectProductsCost",
+    "Cost100": 100
```

Comment:
Let's try to keep the samples as generic as possible - so for this I'd suggest `ProductCost` instead